projects/jsoup/HtmlFuzzer.java

```diff
@@ -15,12 +15,11 @@
 ////////////////////////////////////////////////////////////////////////////////
 
 import com.code_intelligence.jazzer.api.FuzzedDataProvider;
-import org.jsoup.Jsoup;
-import org.jsoup.nodes.Document;
 
+import org.jsoup.Jsoup;
 
-public class JsoupFuzzer {
-	public static void fuzzerTestOneInput(FuzzedDataProvider data) {
-		Document doc = Jsoup.parse(data.consumeRemainingAsString());
```

Comment:
We should strive to keep the fuzz target as simple as possible. I don't think we need to store the result in a local variable, or do we?